### PR TITLE
fixed bug on cordova accessing template via $http on local file:// ,issues with .then().catch(), $templateCache fixes.

### PR DESCRIPTION
**Pull request by MikhailRoot**
+ fixed bug on cordova accessing template via $http on local file://
   we really don't need to load default template via $http service - now using $templateCache directly as we preload it in our module.
+ updated dependencies

### CODE REVIEW

**Comment by interruptr** [comment]
Good job!

File: dist/angular-ui-notification.js [hunk 1]
```diff
@@ -145,7 +160,7 @@ angular.module('ui-notification').provider('Notification', function() {
 
                 setCssTransitions('none');
 
-                angular.element(document.querySelector(args.container)).append(templateElement);
+                angular.element(document.getElementsByTagName('body')).append(templateElement);
```
Comment (juanjoDiaz) [inline dist/angular-ui-notification.js]: Why is this now using the body?
What if I want to use the notification inside a modal? I prefer the flexibility of providing my own selector. 

And just FYI, you can get the body using `document.body`

Comment (MikhailRoot) [inline dist/angular-ui-notification.js]: thank you for insight, but I havent changed this line by myself.   What i've done is dealing with Template loading. I'll just fix it as you've adviced. Thank you.

**Comment by mleanos** [comment]
LGTM. I've verified this fixes the issue.

Would love to have this merged!

@MikhailRoot Thank you for the solution!

**Pull request merged**